test/gtest/ucs/test_type.cc

```diff
@@ -118,9 +118,12 @@ UCS_TEST_F(test_type, pack_float) {
     EXPECT_TRUE(isnan(unpacked));
 
     /* Below min -> min */
-    EXPECT_EQ(UCS_FP8_UNPACK(TEST_LATENCY,
-                             UCS_FP8_PACK(TEST_LATENCY, UCS_BIT(7))),
-              UCS_FP8_UNPACK(TEST_LATENCY, UCS_FP8_PACK(TEST_LATENCY, 15)));
+    for (uint64_t min_val = UCS_BIT(0); min_val <= UCS_BIT(7); min_val <<= 1) {
```

Comment:
Are you sure there's any real point in looping here, rather than just testing a single value?